test/test.cpp

```diff
@@ -46,5 +60,18 @@ int main(int argc, char** argv) {
   test::InitLog();
   test::InitFault();
   testing::InitGoogleTest(&argc, argv);
-  return RUN_ALL_TESTS();
+  int result = 0;
+#ifdef YACLIB_FIBER
+  ::testing::UnitTest::GetInstance()->listeners().Append(new test::MyTestListener());
```

Comment:
Don't like it
Maybe we can calculate how many times was called rand and write it before test starts?
I think on short distance it very predictable.
Don't know, maybe at least 3 option:
1) just random
2) random per test
3) random counts